app/assets/javascripts/projects/edit_fields.js.coffee

```diff
@@ -22,6 +22,80 @@ $ ->
         e.preventDefault()
         $('#fields_form_submit').click()
 
+    # Make table sortable
+    $( '#sortable' ).sortable();
```

Comment:
very minor thing. $('#sortable').sortable(); ... you didn't put spaces in any of the other jquery selectors.